.dependebot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
Added similar to how @gabro updated some of the other repos.

Reply:
cool, but beware: you have a typo in the file name :-)

Reply:
:(

Reply:
Fixed! Thanks for noticing!